test/performance/vectorization/vectorizeYieldingLoopsPragma/vectorZipForLoop.future

```diff
@@ -1,5 +0,0 @@
-bug: vectorize yielding loops should work for zippering as well
```

Comment:
Is there an issue tracking this future that should be closed by this PR?

Reply:
Not that I can see from searching github